apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/service/blockchain/BlockchainImpl.java

```diff
@@ -90,10 +86,9 @@
 
     @Inject
     public BlockchainImpl(BlockDao blockDao,
+                          TransactionService transactionService,
```

Comment:
What is a purpose of using Service class - TransactionService and Repository class TransactionDao simultaneously?

Reply:
removed TransactionDao usage;